client/post-editor/editor-status-label/index.jsx

```diff
@@ -54,20 +57,12 @@ var StatusLabel = React.createClass( {
 	},
 
 	render: function() {
-		var statusClass = 'post-editor-status-label';
+		let statusClass = classNames( 'editor-status-label', {
+			'is-placeholder': ! this.props.post
+		} );
```

Comment:
>This seems a bit odd

I agree. Dropped the placeholder modifier class in a0a8661e8252cdd40511549bd44cec8695b67bab.